.template-lintrc.js

```diff
@@ -7,6 +7,6 @@ module.exports = {
 
   rules: {
     'img-alt-attributes': false,
-    'html-comments': false,
+    'no-html-comments': false,
```

Comment:
I'm seeing `Invalid rule configuration found: no-html-comments` [in travis](https://travis-ci.com/rust-lang/crates.io/jobs/259961925#L344) :-/

Reply:
I forgot to update the linting dependency, it seems!